pkgs/development/libraries/gvfs/default.nix

```diff
@@ -52,21 +52,16 @@ stdenv.mkDerivation rec {
     hash = "sha256-xPbhH8TqqZM/TbjHo0R14GaM6tK//tloZ9Bhvj053aU=";
   };
 
-  patches = [
-    # Hardcode the ssh path again.
-    # https://gitlab.gnome.org/GNOME/gvfs/-/issues/465
-    (fetchpatch2 {
-      url = "https://gitlab.gnome.org/GNOME/gvfs/-/commit/8327383e262e1e7f32750a8a2d3dd708195b0f53.patch";
-      hash = "sha256-ReD7qkezGeiJHyo9jTqEQNBjECqGhV9nSD+dYYGZWJ8=";
-      revert = true;
-    })
-  ];
-
   postPatch = ''
     # patchShebangs requires executable file
     chmod +x meson_post_install.py
     patchShebangs meson_post_install.py
     patchShebangs test test-driver
+
+    # Hardcode the ssh path again, but in a way which works for cross compilation.
+    # https://gitlab.gnome.org/GNOME/gvfs/-/issues/465
+    substituteInPlace daemon/meson.build \
+      --replace '-DSSH_PROGRAM="ssh"' '-DSSH_PROGRAM="${lib.getBin openssh}/bin/ssh"'
```

Comment:
What about the `SFTP backend` part of the patch this replaces? Doesn't that need to be handled anymore?

Reply:
shouldn't be necessary:
```
if enable_sftp
  ssh = find_program('ssh', required: false)
  assert(ssh.found(), 'SFTP backend requested but a ssh client is required')
endif
```

this part of the patch was just a safeguard/diagnostic AFAICT. nothing else in any of the build files references that `ssh` variable (nor does any file in the repo reference the bare `"ssh"`, everything does properly go through `SSH_PROGRAM`)